specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,33 @@
+# Pricing Unit
+
+The Pricing Unit is a measurement unit specified by the provider for determining unit prices associated with the usage or the purchase of a specific resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit. Pricing Unit is complementary to and describes how [Quantity In Pricing Unit](#quantityinpricingunit) is measured. The Pricing Unit is primarily focused on pricing and cost and it should not be confused with the [Usage Unit](#usageunit), which is dedicated to tracking resource and/or service consumption and is often at a lower granularity than the Pricing Unit.
```

Comment:
```suggestion
The Pricing Unit is the provider-specified measurement unit for determining the unit prices associated with the use or purchase of a specific resource or service. Examples include measuring the hours a compute appliance runs, calculating gigabyte-hours for storage appliances, or tallying requests for network appliances or API services. In some cases, providers may opt for block pricing, such as using a million requests as a foundational unit. The Pricing Unit complements and describes how the measurement of [Quantity In Pricing Unit](#quantityinpricingunit) is conducted. The Pricing Unit pertains to pricing and cost determination and should not be confused with the [Usage Unit](#usageunit), which tracks resource and service consumption, typically at a finer granularity than the Pricing Unit.
```

Reply:
Use if more concise. Credit ChatGPT :)

Reply:
We took some bits of those and commited pre-maintainer version to the repo